src/question/tests.py

```diff
@@ -1,3 +1,165 @@
-from django.test import TestCase
+from django.contrib.auth.models import User
+from django.urls import reverse
+from faker import Faker
+from rest_framework import status
+from rest_framework.test import APITestCase
+from taggit.models import Tag
 
-# Create your tests here.
+fake = Faker()
+starting_amount = 5
+added_amount = 5
+
+
+class TagTests(APITestCase):
+    def setUp(self):
+        User.objects.create(username='admin', is_superuser=True, is_staff=True) \
```

Comment:
It think you can use bulk create. https://docs.djangoproject.com/en/2.1/ref/models/querysets/#django.db.models.query.QuerySet.bulk_create